cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -112,15 +112,23 @@ public BaseArgs getArgs() {
 	}
 
 	public BladeSettings getBladeSettings() throws IOException {
-		final File settingsFile;
+		File settingsFile;
 
 		if (WorkspaceUtil.isWorkspace(this)) {
 			File workspaceDir = WorkspaceUtil.getWorkspaceDir(this);
 
 			settingsFile = new File(workspaceDir, ".blade/settings.properties");
+
+			if (!settingsFile.exists()) {
+				settingsFile = new File(workspaceDir, ".blade.properties");
+			}
 		}
 		else {
```

Comment:
Instead of repeating this block, just set a variable called 'settingsBaseDir' and then you can create the file just once.